examples/simulated-light/light.ts

```diff
@@ -2,11 +2,14 @@
 // const uc = require("uc-integration-api");
 //uc.init("light-driver.json");
 
-import uc from "../../index";
+import uc from "../../index.js";
 import * as fs from "fs";
 import * as path from "path";
+import { fileURLToPath } from "url";
 
-const initFilePath = path.resolve(__dirname, "light-driver.json");
+const filename = fileURLToPath(import.meta.url);
+const dirname = path.dirname(filename);
+const initFilePath = path.resolve(dirname, "light-driver.json");
```

Comment:
Keep it simple in the example and the same as in the other examples.
An error is already thrown in `init` if the file cannot be loaded.
One must change into the example directory to start it (see README).

A simple `uc.init("light-driver.json");` is sufficient here.